src/test/java/com/nnaltakyan/core/communications/domain/sentemail/task/SendEmailTaskTest.java

```diff
@@ -0,0 +1,54 @@
+package com.nnaltakyan.core.communications.domain.sentemail.task;
+
+import com.nnaltakyan.core.communications.domain.email.EmailService;
+import com.nnaltakyan.core.communications.domain.sentemail.model.SentEmail;
+import com.nnaltakyan.core.communications.domain.sentemail.service.SentEmailService;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.data.domain.PageImpl;
+import org.springframework.data.domain.Pageable;
+
+import java.util.List;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.*;
+
+
+@ExtendWith(MockitoExtension.class)
+class SendEmailTaskTest {
+
+    private SendEmailTask sendEmailTask;
```

Comment:
@InjectMocks...

Reply:
"Since we have an int type field in our constructor, which is non-mockable, using @InjectMocks will attempt to inject mocks via constructor injection, potentially corrupting the test."

Reply:
Good thank you